test/globalize_test.rb

```diff
@@ -217,4 +217,28 @@ class GlobalizeTest < MiniTest::Spec
       end
     end
   end
+
+  describe 'with locale' do
+    def perform_with_locale(locale)
+      Thread.new do
+        Globalize.with_locale(locale) do
+          sleep 1
```

Comment:
This test is green with or without `sleep` on my machine. That's why I thought that maybe I'm missing something.
I'll use it without `sleep` then, until there's a reason to have it there.